lib/matplotlib/collections.py

```diff
@@ -1149,15 +1149,15 @@ def __init__(self,
         Example: see :doc:`/gallery/event_handling/lasso_demo` for a
         complete example::
 
-            offsets = np.random.rand(20,2)
+            offsets = np.random.rand(20, 2)
             facecolors = [cm.jet(x) for x in np.random.rand(20)]
-            black = (0,0,0,1)
+            black = (0, 0, 0, 1)
 
             collection = RegularPolyCollection(
                 numsides=5, # a pentagon
                 rotation=0, sizes=(50,),
                 facecolors=facecolors,
-                edgecolors=(black,),
+                edgecolors=("k",),
```

Comment:
```suggestion
                edgecolors=("black",),
```
Example code should be as readable as possible :smile:

Reply:
fixed (and deleted the unused definition `black = (0, 0, 0, 1)` just above)